src/doc/trpl/patterns.md

```diff
@@ -23,6 +23,31 @@ match x {
 
 This prints `one`.
 
+There’s one pitfall with patterns: like anything that introduces a new binding,
```

Comment:
We talk about it in the variable bindings section, but this is a question that people ask about with patterns specifically, so I felt it was worth re-calling it out.